src/appMain/hmi_capabilities.json

```diff
@@ -242,6 +242,13 @@
                         "resolutionWidth": 35,
                         "resolutionHeight": 35
                     }
+                },
+                {
+                    "name": "alertIcon",
+                    "imageResolution": {
```

Comment:
Oddly, `imageTypeSupported` is mandatory in the Mobile API, but not the HMI API. It should be included here though

Reply:
included in https://github.com/smartdevicelink/sdl_core/pull/2965/commits/fb7a07c489501a6c933075e1c8d591ff2a3ce744